### PR TITLE
Disables the radiation storm "the air begins to grow warm" message

### PR DESCRIPTION
**Pull request by imsynthex**
## About The Pull Request

Just sets the radiation storm telegraph_message to null

## Why It's Good For The Game

Being able to tell apart a real and fake rad storm completely invalidates fake rad storms

## Testing Photographs and Procedure

Loaded in on a private server and spawned in a rad storm, works as intended

## Changelog
:cl:
tweak: Removed the alert allowing everyone to easily tell apart fake and real radiation storms
/:cl:

### CODE REVIEW

File: code/datums/weather/weather_types/radiation_storm.dm [hunk 1]
```diff
@@ -3,8 +3,7 @@
 	name = "radiation storm"
 	desc = "A cloud of intense radiation passes through the area dealing rad damage to those who are unprotected."
 
-	telegraph_duration = 400
```
Comment (PowerfulBacon) [inline code/datums/weather/weather_types/radiation_storm.dm:6]: Mmm instant rad storms

Comment (imsynthex) [reply code/datums/weather/weather_types/radiation_storm.dm:6]: telegraph_duration is set to 300 by default, so not instant, but slightly faster rad storms i guess

Comment (imsynthex) [reply code/datums/weather/weather_types/radiation_storm.dm:6]: also i had to do testing again to make sure i wasn't going mental

File: code/datums/weather/weather.dm [hunk 2]
```diff
@@ -4,6 +4,7 @@
 	var/name = "space wind"
 	var/desc = "Heavy gusts of wind blanket the area, periodically knocking down anyone caught in the open."
 
+	var/telegraph_enabled = TRUE
```
Comment (PowerfulBacon) [inline code/datums/weather/weather.dm]: Remove this and just set the telegraph_message to null

**Comment by imsynthex** [comment]
ok all done 🙂

**Review by PowerfulBacon — APPROVED** [review]

**Comment by Archanial** [comment]
Update PR body to reflect the changes please

**Pull request merged**